### ISSUE TITLE
Contribution guide

### ISSUE THREAD

**Issue by icebob** [icebob/vue-form-generator#165]
I see vfg is more familiar day-by-day. So maybe we need a contribution guide some base rules...
E.g.
* don't commit dist files
* don't throw errors in source codes
* if modify a function, update tests, if add new function cover with new tests
* ... etc

What do you think?

**Comment by cristijora** [comment]
I personally agree with everything except dist giles. Even though I ddin't commit them so far, if they are commited you have the advantage of simply putting those in a fiddle and testing out stuff like I did with the string validators.

**Comment by icebob** [comment]
Rightful, but if there are two PRs with dist files, will be problem with merging.

**Comment by jmverges** [comment]
Hi there guys,

yes, a contribution guide is always appreciated. To not commit dist files, I think is correct. Maybe this should be in the gitignore and use the force in each release. On the other hand, if you adopt this way, could be useful to have a release each 2 weeks if there are any changes.

My two cents ;)

**Comment by lionel-bijaoui** [comment]
Yes, can we also agree on a coding style ? Because the sometime lack of curly braces caused many mistakes...
```
// don't do this please !
if (true)
    console.log('hello');

// do this please !
if (true) {
    console.log('hello');
}
```
